migrate-mongo-config.js

```diff
@@ -0,0 +1,34 @@
+// In this file you can configure migrate-mongo
+
+const config = {
+  mongodb: {
+    // TODO Change (or review) the url to your MongoDB:
+    url: "mongodb://root:root@127.0.0.1:27017/admin",
+
+    // TODO Change this to your database name:
+    databaseName: "admin",
```

Comment:
@JustinaMary I guess that for this we need to introduce a second environment variable MONGO_DB containing the database name which is environment specific.

Reply:
Also add this variable to the .env.example and .env.test files

Reply:
Done